### PR TITLE
Feature/316 measure create edit

### PR DESCRIPTION
**Pull request by lkleisa**

### CODE REVIEW

File: frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -19,6 +20,14 @@ const routes: Routes = [
   },
   { path: 'teams', component: TeamListComponent },
   { path: 'keyresults/:id', component: KeyResultOverviewComponent },
+  {
+    path: 'keyresults/:keyresultId/measure/new',
+    component: MeasureFormComponent,
+  },
+  {
+    path: 'keyresults/:keyresultId/measure/:measureId',
```
Comment (peggimann) [inline frontend/src/app/app-routing.module.ts:28]: use edit, like in the other routes

File: frontend/src/app/measure/measure-form/measure-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+export class MeasureFormComponent implements OnInit {
+  measure$!: Observable<Measure>;
+
+  measureForm = new FormGroup({
+    value: new FormControl<number>(0, [
+      Validators.required,
+      Validators.pattern('^[0-9]*$'),
+    ]),
+    measureDate: new FormControl<Date>(new Date(), [Validators.required]),
+    changeInfo: new FormControl<string>('', [Validators.required]),
```
Comment (peggimann) [inline frontend/src/app/measure/measure-form/measure-form.component.ts:27]: changeInfo and Initiatives are not required.

File: frontend/src/app/measure/measure-form/measure-form.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+
+      expect(mockMeasureService.saveMeasure).toHaveBeenCalledTimes(1);
+      expect(mockMeasureService.saveMeasure).toHaveBeenCalledWith(
+        receivedEditedMeasure,
+        false
+      );
+    });
+  });
+
+  describe('Create new Measure', () => {
```
Comment (peggimann) [inline frontend/src/app/measure/measure-form/measure-form.component.spec.ts:273]: only use required validators

**Review by peggimann — CHANGES_REQUESTED** [review]

File: frontend/src/app/app-routing.module.ts (hunk 1 above)
Comment (MarcEgliP) [reply frontend/src/app/app-routing.module.ts:28]: Edit is now in path

File: frontend/src/app/measure/measure-form/measure-form.component.ts (hunk 2 above)
Comment (MarcEgliP) [reply frontend/src/app/measure/measure-form/measure-form.component.ts:27]: Die initiatives und das measureDate sind nun nicht mehr required

File: frontend/src/app/measure/measure-form/measure-form.component.spec.ts (hunk 3 above)
Comment (MarcEgliP) [reply frontend/src/app/measure/measure-form/measure-form.component.spec.ts:273]: Validators were changed in tests

**Review by peggimann — APPROVED** [review]

**Pull request merged**